zadanie 10.5/js/scripts.js

```diff
@@ -0,0 +1,16 @@
+function Button(text) {
+	this.text = text || 'Hello';
+}
+Button.prototype = {
+	create: function() {
+	var self = this;
+	this.$element = $('<button>');
+	this.$element.text(this.text);
+	this.$element.click(function() {
+		alert(self.text);
+		this.$element.appendTo($('body'));
+	});
+	var btn1 = new Button('Hello!');
+		btn1.create(); 
```

Comment:
te dwie linie powinny byc poza prototypem - tak po prostu wywolujesz to, co napisales